styles/globals.css

```
@@ -207,3 +207,49 @@
+    -webkit-text-fill-color: inherit !important;
+    -webkit-box-shadow: none !important;
+    transition: background-color 50000s ease-in-out 0s !important;
+  }
+  
+  /* Make isolated inputs completely disconnected from forms */
+  [data-form-isolated] input {
+    -webkit-autofill: none !important;
+    -webkit-autocomplete: off !important;
+    -webkit-appearance: textfield !important;
+    -moz-appearance: textfield !important;
+    form: none !important;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)